packages/contracts-bedrock/contracts/test/ProxyAdmin.t.sol

```diff
@@ -164,19 +164,26 @@ contract ProxyAdmin_Test is Test {
         vm.prank(alice);
         admin.changeProxyAdmin(_proxy, address(128));
 
-        // The proxy is not longer the admin and can
+        // The proxy is no longer the admin and can
         // no longer call the proxy interface except for
-        // the ResolvedDelegate type which anybody can call
-        // the admin interface
-        if (proxyType != ProxyAdmin.ProxyType.ResolvedDelegate) {
-            vm.expectRevert();
+        // the ResolvedDelegate type on which anybody can
+        // call the admin interface.
+        if (proxyType == ProxyAdmin.ProxyType.ERC1967) {
+            vm.expectRevert("Proxy: implementation not initialized");
```

Comment:
A change below in the ProxyAdmin uses a high level syntax to make a static call, which also bubbles up the error message, so can't just use a generic `expectRevert()` here. 

(I was surprised that `expectRevert()` fails in the case there there is an error message).